packages/aws-cdk/lib/api/hotswap/ecs-services.ts

```diff
@@ -119,11 +119,11 @@ export async function isHotswappableEcsServiceChange(
 
         // Step 3 - wait for the service deployments triggered in Step 2 to finish
         // configure a custom Waiter
-        (sdk.ecs() as any).api.waiters.deploymentToFinish = {
-          name: 'DeploymentToFinish',
+        (sdk.ecs() as any).api.waiters.deploymentCompleted = {
+          name: 'DeploymentCompleted',
           operation: 'describeServices',
-          delay: 10,
-          maxAttempts: 60,
+          delay: 6,
+          maxAttempts: 100,
```

Comment:
can you briefly explain the reason for changing these?

Reply:
Ah, I just reminded myself from my commit message for this change as well: the `FAILED` intermediate state for the deployment in the ECS API exists for a reasonably short time. So this change also ensures that we catch that intermediate `FAILED` state